addons/payment_adyen/tests/test_adyen.py

```diff
@@ -2,17 +2,20 @@
 
 from unittest.mock import patch
 
+from werkzeug.exceptions import Forbidden
+
 from odoo.exceptions import UserError
 from odoo.tests import tagged
 from odoo.tools import mute_logger
 
 from odoo.addons.payment import utils as payment_utils
-
-from .common import AdyenCommon
+from odoo.addons.payment.tests.http_common import PaymentHttpCommon
+from odoo.addons.payment_adyen.controllers.main import AdyenController
+from odoo.addons.payment_adyen.tests.common import AdyenCommon
 
 
 @tagged('post_install', '-at_install')
-class AdyenForm(AdyenCommon):
+class AdyenTest(AdyenCommon, PaymentHttpCommon):
```

Comment:
might make sense to merge http & non http commons in payment now :thinking:

Reply:
Already planned.